### ISSUE TITLE
[Feature request] CSS Modules support for "Find All References" / "Peek References"

### ISSUE THREAD

**Issue by theinfinit** [pucelle/vscode-css-navigation#90]
Hi Pucelle, 
how hard would it be to add CSS Modules support for "Find All References" and "Peek References"?

Example of CSS Module reference in Vue:
```vue
<!-- App.vue -->
<script setup lang="ts">
import s from './styles.module.css'
</script>

<template>
  <div :class="s.h_screen"></div>
</template>
```

And definition of `styles.module.css`
```css
/* styles.module.css */
.h_screen {
  height: 100vh;
}
```

Currently "Find All References" and "Peek References" options work only for standard CSS classes, but how hard would it be to implement the same functionality for CSS Modules?

Cheers ✌️

**Comment by pucelle** [comment]
Let me check....

**Comment by pucelle** [comment]
Sorry, this plugin has no plan to directly process any "JavaScript Expression Like" syntax.

**Comment by pucelle** [comment]
Will be released in coming version 1.14.1

**Comment by theinfinit** [comment]
Oh, that's fantastic, Pucelle! Thank you for your hard work. 

Have a great day 😇
